### PR TITLE
Increase the tx timeout in testTransactionEnlistment

### PR DESCRIPTION
**Pull request by dazavala**
Transactions may timeout in `SimpleServlet.testTransactionEnlistment()` due to slow test servers and a big trace spec. Bump the transaction timeout to 5 seconds for now.

### CODE REVIEW

**Comment by dazavala** [comment]
#build
#libby

**Comment by dazavala** [comment]
@anjumfatima90 I'll notify for review once this change passes the InstantOn and and personal builds.

**Comment by dazavala** [comment]
@anjumfatima90 Liberty and Liberty InstantOn builds passed over the weekend.  This fix is ready to merge pending approval.

**Review by anjumfatima90 — APPROVED** [review]

**Pull request merged**
